src/core/toolbox/index.js

```diff
@@ -22,6 +23,7 @@ export default {
         [ 'detach', DetachBehavior ],
         [ 'element.hover', HoverBehavior ],
         [ 'element.out', OutBehavior ],
+        [ 'keypress', KeypressBehavior ],
```

Comment:
An Idea to improve the KeyBehaviors is to define it like this:

`[ [ 'esc' ], KeypressBehavior ],`

Reply:
:+1: